spec/interactors/login_user/generate_jwt_spec.rb

```diff
@@ -0,0 +1,29 @@
+require "spec_helper"
+
+RSpec.describe LoginUser::GenerateJwt do
+  subject(:result) { described_class.call(user: user) }
+
+  let(:user) { create(:user, raw_user_attributes) }
+  let(:fake_codec) { instance_double(JwtCodec) }
+  let(:raw_user_attributes) do
+    { email: "jwt_user@example.com", full_name: "Simon Petrikov" }
+  end
+  let(:user_attributes) { raw_user_attributes.merge(id: user.id) }
+
+  before do
+    allow(JwtCodec).to receive(:new).and_return(fake_codec)
+    allow(fake_codec).to receive(:encode)
+      .with(user_attributes).and_return("8")
```

Comment:
In my opinion, we shouldn't stub token here. I think we should only check that `JwtCodec` received a new method, and then its received encode method with a correct payload.
And we should check that token is not nil.